### ISSUE TITLE
docs(0004): deprecate `ledgerInfo.precision`?

### ISSUE THREAD

**Issue by michielbdejong** [interledger/rfcs#179]
Usually, ledger plugins will announce precision 19 in their response to `plugin.getInfo()`. In the [current spec](https://github.com/interledger/rfcs/blob/master/0004-ledger-plugin-interface/0004-ledger-plugin-interface.md#class-ledgerinfo) it's not very clear whether this means "this ledger can process transactions with amounts up to 19 decimal digits", or, "this ledger can represent account balances of up to 19 decimal digits".

If `ledgerInfo.precision` refers to transactions, then announcing a value >19 is useless, since ILP defines the maximum `amount` of a transaction to be UInt64. Announcing a value <19 means announcing that some ILP payments will be rejected by this ledger for reason "amount too high". In this case, this error response would have to be implemented anyway, and I would argue that announcing a transaction precision <19, in the hope that senders/connectors will refrain from trying to send very big transactions, is needlessly complex, given that even with this mechanism, the error response mechanism would still be needed.

If `ledgerInfo.precision` refers to account balance, then I also don't see the point of programmatically announcing it. A ledger administrator can attract customers by saying "our ledger allows account balances for the super-rich", but I can't see a situation in practice where a receiver would do an extra check before inviting payments to their account, just to make sure the ledger's precision is high enough to still support the intended new balance after getting paid. Also, the ledger would still have to respond with an error if completing a payment would mean someone's account balance would get too high to represent.

In other words, if we don't know in what situation someone would want to check `ledgerInfo.precision` programmatically, then maybe we should just deprecate it, and simplify the plugin interface?

**Comment by michielbdejong** [comment]
Done.
